### ISSUE TITLE
for the love, please stop using cloudflare

### ISSUE THREAD

**Issue by mdik** [torontocrypto/torontocrypto.github.io#12]
Especially as you are hosted on github.io, where is the point?

**Comment by louipc** [comment]
On Sun 03 May 2015 16:39 -0700, Malte wrote:
> Especially as you are hosted on github.io, where is the point?

Hi can you describe the issue in greater detail?
Seems like cloudflare is injecting some scripts into the web page.

darknetj says they are using cloudflare for an ssl certificate.
Seems to me like this is not a great solution. Got any ideas?

**Comment by darknetj** [comment]
There is no quick way to setup SSL through Github deployment on a custom domain.

Github issues for SSL on custom domains: https://github.com/isaacs/github/issues/156

https://konklone.com/post/github-pages-now-sorta-supports-https-so-use-it

Until we move to a different deployment process (or Github gets their act together)
we'll disable the Tor captcha on Cloudflare.

**Comment by mdik** [comment]
The captcha still pops up.

But I also think that that's a bit beside the point: TLS through Cloudflare is not really TLS. So why don't you drop the act?

Either set up something like a Raspberry Pi with DynDNS from home or rent a cheap v-server to host torontocrypto.org. There, you can deploy your own certificates. If you need any assistance with one of the steps, please let me know.

**Comment by dmix** [comment]
I tweaked the security settings to a lower level, so hopefully it will stop blocking tor vistors.

Cloudflare was a temporary solution to get SSL. We were collecting emails on our site and needed something up fast. Github is great for getting something up fast, but their lack of proper SSL support is a problem. 

Although both ends have it: Cloudflare (SSL) <> Github.io (SSL) so it's not really that bad.

We should plan to move to a host that supports full SSL certs at some point. Plus we have plans to add a proper onion route.

**Comment by mdik** [comment]
Yes, now I can see it from Tor. Would you mind making a screenshot of the Cloudflare settings page, so that we could distribute a little "How to unblock Tor" for other Cloudflare users?
